frontend/shared/src/constants.ts

```diff
@@ -10,3 +10,11 @@ export const POSTAL_CODE_REGEX = /^\d{5}$/;
 export const NAMES_REGEX =
   /^[\w',.ÄÅÖäåö-][^\d!#$%&()*+/:;<=>?@[\\\]_{|}~¡¿÷ˆ]+$/;
 export const CITY_REGEX = /^[ A-Za-zÄÅÖäåö-]+$/;
+
+export const EMAIL_REGEX =
+  // eslint-disable-next-line security/detect-unsafe-regex
+  /^(([^\s"(),.:;<>@[\\\]]+(\.[^\s"(),.:;<>@[\\\]]+)*)|(".+"))@((\[(?:\d{1,3}\.){3}\d{1,3}])|(([\dA-Za-z-]+\.)+[A-Za-z]{2,}))$/;
+
+// eslint-disable-next-line security/detect-unsafe-regex
+export const DATE_UI_REGEX = /^(?:\d{1,2}.){2}\d{4}$/;
+export const DATE_BACKEND_REGEX = /^\d{4}-\d{2}-\d{2}$/;
```

Comment:
I used in my PR more thorough versions for validating dates inspired by [this answer on Stack Over Flow](https://stackoverflow.com/a/22061879/9441362) so I didn't have to disable the security rule.
https://github.com/City-of-Helsinki/yjdh/blob/71f2fb70942dc842c7fcbaeaf198ec505cf36559/frontend/shared/src/utils/date.utils.ts#L40
https://github.com/City-of-Helsinki/yjdh/blob/71f2fb70942dc842c7fcbaeaf198ec505cf36559/frontend/shared/src/utils/date.utils.ts#L44

Reply:
I started to use these 👍